bin/generate-version-file.js

```diff
@@ -15,8 +15,8 @@ const path = require('path');
 function checkEnvironment() {
   const isRunningInCircleCi = process.env.CIRCLECI === 'true';
   if (!isRunningInCircleCi) {
-    throw new Error(
-      'This script will not work correctly outside of a CircleCI environment.'
+    console.warn(
+      'Warning: This script will not work correctly outside of a CircleCI environment.'
     );
   }
```

Comment:
Thanks for the example! Yeah it could be simpler to use a shell script here.
One advantage of using node is that it's portable (notably to Windows), and so this is an easier path for contributors.
Of course if we build inside the docker container, then it's less of a problem.

Reply:
I tried using a shell script, it's included in this PR as an experimentation, but I don't like it so much and I think I'll keep the node-based script. (see https://github.com/firefox-devtools/profiler-server/pull/10/files#r340254501)